sdk/servicebus/service-bus/package.json

```diff
@@ -143,6 +143,7 @@
     "eslint": "^6.1.0",
     "esm": "^3.2.18",
     "glob": "^7.1.2",
+    "https-proxy-agent": "^5.0.0",
```

Comment:
This is used only in samples, and the package.json file in samples do list it as a dependency. Why do we need it here?

Reply:
So we can do npm run build:samples

Reply:
Logged an issue https://github.com/Azure/azure-sdk-for-js/issues/12152